mlops_utilities/actions.py

```diff
@@ -177,64 +201,101 @@ def compare_metrics(sagemaker_client,
+    :param instance_type
+    :param instance_count
     :param endpoint_name:
     :param data_capture_config: config for inference data capture
     :param model_statistics_s3_uri: s3 bucket which contains evaluation metrics
     :param metric: path to metric value in `model_statistics_s3_uri`
     :param dryrun: is 'True' in a case of testing
     :return:
     """
-    endpoint_config_description = sagemaker_client.describe_endpoint_config(EndpointConfigName=endpoint_name)
-    model_name = endpoint_config_description["ProductionVariants"][0]["ModelName"] if dryrun is False else 'model'
-    require_update = metric is None or (metric is not None and compare_metrics(sagemaker_client,
-                                                                               endpoint_config_description,
-                                                                               model_statistics_s3_uri,
-                                                                               metric))
+    endpoint_config_description = sagemaker_client.describe_endpoint_config(
+        EndpointConfigName=endpoint_name
+    )
+    model_name = (
+        endpoint_config_description["ProductionVariants"][0]["ModelName"]
+        if not dryrun
+        else "model"
+    )
+    require_update = metric is None or (
+        metric is not None
+        and compare_metrics(
+            sagemaker_client,
+            endpoint_config_description,
+            model_statistics_s3_uri,
+            metric,
+        )
+    )
 
     if require_update:
         predictor = Predictor(endpoint_name=endpoint_name)
-        predictor.update_endpoint(initial_instance_count=instance_count,
-                                  instance_type=instance_type,
-                                  model_name=model_name)
+        predictor.update_endpoint(
+            initial_instance_count=instance_count,
+            instance_type=instance_type,
+            model_name=model_name,
+        )
         predictor.update_data_capture_config(data_capture_config)
     else:
-        logger.info(
-            "Current endpoint is not updated because the new model have worse quality than current deployed model"
-        )
-
-
-def create_endpoint(model_package_arn: str,
-                    sagemaker_session: Session,
-                    instance_count: int,
-                    instance_type: str,
-                    endpoint_name: str,
-                    data_capture_config: DataCaptureConfig,
-                    role: str) -> NoReturn:
+        if logger.isEnabledFor(logging.INFO):
+            logger.info(
+                "Current endpoint is not updated because the new model have worse quality than current deployed model"
+            )
+
+
+def create_endpoint(
+    model_package_arn: str,
+    sagemaker_session: PipelineSession,
+    instance_count: int,
+    instance_type: str,
+    endpoint_name: str,
+    data_capture_config: DataCaptureConfig,
+    role: str,
+) -> NoReturn:
```

Comment:
`NoReturn` is used for cases where function does not return normally, meaning when it is expected to produce an exception.